_posts/2022-03-05-gs_00006_file_permissions.md

```diff
@@ -0,0 +1,216 @@
+also can see the owner and group of the file which is in this case both set to
+_bitsandvolts_.
+
+Now, we want to show how to change the permission so that nobody except the
+owner is allowed to read the test file.
+
+    $ chmod go= test
+
+How does this command work? _g_ stands for group and _o_ for other. In this
+case, you don't set any permissions for the group and others for our test file
+because there is no value behind the equal sign. Check if the command worked
+with:
+
+    $ ls -l
+
+When you try to run the file by
+
+    $ ./test
+
+you will get a _permission denied_ error. Why? Of course, because it does not
+have the execute permission. So, let's add it! You have multiple options to do
+it. You can add the permission by:
+
+    $ chmod u+x test
+
+<div class='note-normal' markdown="1">
+_Note: u stands for user and means the owner of the file. In case,
+you want to remove a permission you can use the minus sign._
+</div>
+
+Or, you can set the complete user permissions by:
+
+    $ chmod u=rwx test
+
+Now, you can run the test file without error. Of course, it does nothing since
+it is empty:
+
+    $ ./test
+
+One tiny note is missing: Who is allowed to change the file permissions? It
+would be stupid if students could change the file permissions of the exam
+themselves, wouldn't it? Only the owner of the file and the **superuser** can
+change it. We talk about the superuser later, but it's basically the owner of
+the machine who can do whatever he wants.
+
+That's it! Easy, isn't it?
+
+# Set permissions using numeric values
+
+Very often there is more than just one way to achieve a goal. There is another
+syntax using numeric values that is commonly used for file permissions. If you
+want to set all permissions (read, write and execute) for the file's owner and
+no permissions for everyone else you can also use the following command:
+
+    $ chmod 700 test
+
+How does this work? Behind the previously used `chmod` command, you see three
+digits. The first represents the owner permissions, the second the permissions
+for the group, and the last for all others. The following table shows the value
+for each permission.
+
+| Permission | Value |
+| ---------- | ----- |
+| Execute    | 1     |
+| Write      | 2     |
+| Read       | 4     |
+
+If you want to combine multiple permissions you just add up the values.
+execute + read + write is represented by 1 + 2 + 4 = 7. The other two values are
+zero since we do not have any permissions for the group and others.
+
+<div class='note-exercise' markdown="1">
+Exercise:
+
+What would be the command for our exam example? Teacher (owner) has read + write
+permissions, participating students (group) has read and everyone else has no
+permissions.
+
+</div>
+
+The solution is:
+
+    $ chmod 640 test
+
+You can decide for yourself which way you prefer. The way from the previous
+section is maybe easier to remember, but it’s completely up to you.
```

Comment:
The last part of this sentence is kind of redundant, as we already said something similar in the previous sentence.

Maybe we can write something like
```suggestion
section is maybe easier to remember, but both achieve the same goal.
```